editor/scaffolds/grid/columns/select-column.tsx

```
@@ -58,16 +73,21 @@ export const SelectColumn: CalculatedColumn<any, any> = {
     // eslint-disable-next-line react-hooks/rules-of-hooks
     const [isRowSelected, onRowSelectionChange] = useRowSelection();
     const rootprops = useCellRootProps(row.__gf_id, column.key);
     return (
       <CellRoot {...rootprops} className="group">
-        <SelectCellFormatter
+        <SelectColumnCell
           aria-label="Select"
           tabIndex={props.tabIndex}
           value={isRowSelected}
-          row={props.row}
+          expandable={!!!row && !!!onExpandClick ? true : false}
```

Biome (1.9.4)

[error] Unnecessary use of boolean literals in conditional expression.

Simplify your code by directly assigning the result without using a ternary operator.
If your goal is negation, you may use the logical NOT (!) or double NOT (!!) operator for clearer and concise code.
 Check for more details about NOT operator.
Unsafe fix: Remove the conditional expression with


(lint/complexity/noUselessTernary)